### DISCUSSION TITLE
MSIX Installer

### DISCUSSION THREAD

**Discussion by SoCuul** [rocksdanister/lively#1892] [Ideas]
It would be amazing if you could distribute the installer as the appx/msix installer along with the exe. As of right now, it's the only way to install Lively on a non-admin non-msstore windows install.

**Comment by rocksdanister** [comment]
Store link
https://github.com/rocksdanister/lively/wiki/Differences-Between-Distributions#windows-store

Download MSIX
https://store.rg-adguard.net/

Powershell instruction for old version if no AppX installer present
https://github.com/rocksdanister/lively/issues/361#issuecomment-1072357161

**Reply by SoCuul** [reply]
Is there any reason why you don't provide any direct msix downloads?

**Reply by rocksdanister** [reply]
I don't have signing certificate currently.

**Reply by SoCuul** [reply]
Ahh that makes sense. Would you consider packaging dotnet desktop into the inno setup package?

**Reply by rocksdanister** [reply]
Why not just install the MSIX? its packaged and signed by Store.

Lively is split into separate programs (UI, Core, Video player..) if I include runtime into each program it will affect the size too much.

Microsoft Powertoys is also designed in a similar way, recently I think they started using a single runtime folder.. not sure how.
